kroxylicious/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -253,7 +267,12 @@ public void initiateConnect(HostPort remote, List<FilterAndInvoker> filters) {
         if (logFrames) {
             pipeline.addFirst("frameLogger", new LoggingHandler("io.kroxylicious.proxy.internal.UpstreamFrameLogger"));
         }
-        addFiltersToPipeline(filters, pipeline);
+        final List<FilterAndInvoker> filterAndInvokers = new ArrayList<>();
```

Comment:
we've already got the KafkaProxyInitializer adding internal filters. would the responsibility to add this one better belong there?

Reply:
Missed that, moved it into the Initializer